speech-to-text/webaudio-l16-stream.js

```diff
@@ -80,7 +80,10 @@ WebAudioL16Stream.prototype.downsample = function downsample(bufferNewSamples) {
     buffer = bufferNewSamples;
   }
 
-  // downsampling variables
+  // you can't just drop or average samples because it won't sound right; it's the same idea as anti-aliasing jagged
+  // edges in game graphics. need to interpolate between samples rather than just retaining and discarding samples
+  // (which introduces horrible jitter noise during non-integer downsampling). Then, need to low-pass filter before or
+  // in conjunction with the interpolation to the lower sample rate, to avoid aliasing.
```

Comment:
@nfriedly 

> I'd rather omit that then have an incorrect description there.

which part do you want omitted? this? :

`Then, need to low-pass filter before or
  // in conjunction with the interpolation to the lower sample rate, to avoid aliasing.`